ESOAsg/scripts/fix_sphere.py

```diff
@@ -99,12 +99,17 @@ def main(args):
                                                                                            suffix=suffix_string + '_WL')
     else:
         output_whitelight_files = [None] * len(input_fits_files)
-    '''
-    # reference
     if args.referenc is not None:
         reference = str(args.referenc[0])
     else:
-        reference = str(' ')
+        reference = str('10.1051/0004-6361/201832973') 
+        # Galicher, R., Boccaletti, A., Mesa, D., et al. (2018) "Astrometric and 
+        #photometric accuracies in high contrast imaging: The SPHERE speckle 
+        #calibration tool (SpeCal)," A&A, 615, A92 - 2018A&A...615A..92G 
+        #https://ui.adsabs.harvard.edu/abs/2018A%26A...615A..92G/abstract
+
+    '''
+    # reference
```

Comment:
I would set reference as global variable in the script, i.e. just after: SUPPORTED_INSTRUMENTS = [..]
This will help in case of future modification of the script.

Also, please follow the PEP8 style guide of having a space after '#' for coments